packages/suite/src/views/dashboard/DashboardPromoBanner/DashboardPromoBanner.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+import { Feature, selectFeatureConfig } from '@suite-common/message-system';
+import { EventType, analytics } from '@trezor/suite-analytics';
+
+import { useSelector } from 'src/hooks/suite';
+
+import { TrezorExpertBanner } from './TrezorExpertBanner';
+import { DashboardBannerType } from './dashboardBannerTypes';
+import { selectIsTEXDashboardPromoBannerShown } from '../../../reducers/suite/suiteReducer';
+
+export const DashboardPromoBanner = () => {
+    const [isVisible, setIsVisible] = useState(true);
+
+    const shouldShowTEXDashboardPromoBanner = useSelector(selectIsTEXDashboardPromoBannerShown);
+
+    const promoBanner = useSelector(state =>
+        selectFeatureConfig(state, Feature.dashboardPromoBanner),
+    );
+
+    const currentBanner = (promoBanner?.visibleBanner ?? null) as DashboardBannerType;
```

Comment:
I couldn't resist to try and find more type-safe way:
```suggestion
    const promoBannerPayload = promoBanner?.visibleBanner;
    const currentBanner = isDashboardBannerType(promoBannerPayload) ? promoBannerPayload : null;
```

with this in `dashboardBannerTypes.ts`
```
export const isDashboardBannerType = (bannerType: unknown): bannerType is DashboardBannerType =>
    bannerType === null ||
    (typeof bannerType === 'string' && isArrayMember(bannerType, dashboardBannerTypes));

```

Reply:
Much better, thanks!